src/main/resources/hazelcast-config-schema.json

```diff
@@ -1206,6 +1206,158 @@
         }
       },
       "description": "When you want to use a Hazelcast member as a lite member, set this element's \"enabled\" attribute to true in that member's YAML configuration. Lite members do not store data, do not have partitions and are used mainly to execute tasks and register listeners."
+    },
+    "CPSubsystem": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "cp-member-count": {
+          "default": 0,
+          "anyOf": [
+            {
+              "const": 0
+            },
+            {
+              "type": "integer",
+              "minimum": 3
+            }
+          ],
+          "description": "Number of CP Members to initialize the CP Subsystem. The CP subsystem is enabled when a positive value is set. After the CP subsystem is initialized successfully, more CP members can be added at run-time and number of active CP members can go beyond the configured CP member count. Number of CP members can be smaller than total size of the Hazelcast cluster. For instance, you can run 5 CP members in a 20-member Hazelcast cluster. If set, must be greater than or equal to \"group-size\"."
+        },
+        "group-size": {
+          "type": "integer",
+          "minimum": 3,
+          "maximum": 7,
+          "not": {
+            "multipleOf": 2
```

Comment:
I think we can highlight this bit in the presentation. More advanced than usual validation and quite useful.